src/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+spring:
+  datasource:
+    initialization-mode: always
+    password: ${DATABASE_PASSWORD}
+    platform: postgresql
+    url: jdbc:postgresql://localhost:5432/payment
+    username: ${DATABASE_USERNAME}
+  jpa:
+    database-platform: org.hibernate.dialect.PostgreSQL10Dialect
+    hibernate:
+      ddl-auto: update
+    show-sql: true
```

Comment:
it's ok to locally log SQL, but not in the environment. Such configs aren't committed usually

Reply:
Was removed